linux_os/guide/services/obsolete/r_services/no_user_host_based_files/ansible/shared.yml

```diff
@@ -22,6 +22,7 @@
   ansible.builtin.set_fact:
     user_shosts_files: '{{ user_shosts_files | union(item.stdout_lines) | list }}'
   loop: "{{ result_found_shosts_files.results }}"
+  when: result_found_shosts_files is not skipped and item is not skipped
```

Comment:
Similar to my previous comment in `no_host_based_files`.

Reply:
all the conditions about skipped, or not skipper rules is done in order to make sure that when ansible is run in check mode it will not fail. In that case most of the system/cmd steps are skipped, and the patches here are basically to prevent failures to run the ansible playbook.

Reply:
It is not necessary in this rule. It should work fine even in check mode.